services/bettersocial/chat/updateBetterChannelMembers.js

```diff
@@ -2,6 +2,7 @@ const _ = require('lodash');
 
 const UsersFunction = require('../../../databases/functions/users');
 const {sequelize} = require('../../../databases/models');
+const BetterSocialConstantListUtils = require('../../../services/bettersocial/constantList/utils');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new import `BetterSocialConstantListUtils` is used later in the code to generate random emoji and color for anonymous users when no previous information is available. This seems like a reasonable addition.
```diff
+ const BetterSocialConstantListUtils = require('../../../services/bettersocial/constantList/utils');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->